### DISCUSSION TITLE
The download Apk of the latest update of Osu! Lazer is the wrong version

### DISCUSSION THREAD

**Discussion by KeiroFrezze** [ppy/osu#21862] [Q&A] [answered]
I report that when I try to download the latest update of osu! Lazer (2022.1227.0) ended up downloading the previous version (2022.1208.0).
I would really appreciate it if you could fix it :)🤝

**Comment by frenzibyte** [comment, marked as answer]
Please avoid opening discussion threads in non-English language. Also if this is about latest release having the APK from previous version then that's intended for a bit, we're still working towards deploying an APK for the new release.

**Reply by KeiroFrezze** [reply]
oh ok thanks and sorry

**Reply by ghost** [reply]
when does the apk file usually update to the correct latest version?

**Reply by frenzibyte** [reply]
The updated APK has since been published. Check out latest release again.
